meta-freescale-3rdparty-mbl/recipes-bsp/atf/atf-imx6ul-des0258-mbl_git.bb

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+inherit mbl-artifact-names
+require recipes-bsp/atf/atf.inc
+
+PLATFORM = "picopi"
```

Comment:
More PICO board references?

Reply:
We do unfortunately need to select something here - would an ARM VFP be acceptable ?
If not I can look into making a dummy ATF configuration for this board.